tests/integration/attributes/test_transaction_non_web_clm.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*DESCRIPTION
+In a non-web transaction that has no user defined functions, code level metrics (CLM)
+should return the filename as the function name (because we are instrumenting the file)
+and lineno 1.
+The agent should include CLM agent attributes in error traces, error
+events, analytic events and span events.
```

Comment:
Is this test doing what the description say? I don't see CLM attributes in expected sections...

Reply:
Please see: [49a22d7](https://github.com/newrelic/newrelic-php-agent/pull/585/commits/49a22d7a6abb50633602e0320f09f65ac8d0384a)